mantidimaging/gui/windows/spectrum_viewer/view.py

```diff
@@ -134,6 +134,11 @@ def on_data_in_table_change() -> None:
         header.setSectionResizeMode(1, QHeaderView.ResizeToContents)
         header.setSectionResizeMode(2, QHeaderView.ResizeToContents)
 
+    def show(self):
+        self.main_window.spectrum_viewer = self
```

Comment:
Does this not get set in `MainWindowView.show_filters_window()`

Reply:
Removed `self.main_window.spectrum_viewer = self` but we still need `self.activateWindow()` to be added to show here for the window focus to be loaded above the main window when called.